themes/theme-default/src/client/styles/code.scss

```diff
@@ -1,8 +1,8 @@
 @import '_variables';
 
 // Safari only support integer pixel value for line-height
```

Comment:
If  the highlight lines and line numbers are rendered on the actual code line, then we can remove this comment. Because every highlight line and line numbers will be beside the lines without misalign after hundreds of lines